bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/regexp/TreeBuilder.java

```diff
@@ -106,6 +109,7 @@ private RegExpTerm readRegTerm() {
             default:
                 // Read chars in ReLiteralChar, . or ReEscape.
                 reAtom = readRegChars();
+                break;
```

Comment:
We don't need a break statement in here, since it is the last check in the switch case.